arpa-exporter/src/worker.py

```diff
@@ -89,6 +89,9 @@ def build_zip(fh: typing.BinaryIO, source_uploads: typing.Iterator[UploadInfo])
 
             try:
                 archive.write(source_path, arcname=upload.path_in_zip)
+            except FileNotFoundError:
+                entry_logger.exception("source file not found")
+                continue
```

Comment:
I'd log this as a less-noisy `error` rather than an `exception` since the traceback/execution info isn't really informative and the resulting log can be a bit more succinct:
```suggestion
            except FileNotFoundError as e:
                entry_logger.error("source file not found", error=e, )
                continue
```